client/features/auth/CreateAccount.jsx

```diff
@@ -84,7 +84,7 @@ class CreateAccount extends React.Component {
     }
 
     return (
-      <div id="CreateAccount" onKeyPress={ this.handleKeyPress }>
+      <div id="CreateAccount" onKeyPress={this.handleKeyPress}>
```

Comment:
Where is the function this.handleKeyPress? I can't locate it here.

Reply:
It's up there above the render method somewhere. Did you expand the file? Usually it just shows you snippets that were modified.